### ISSUE TITLE
Add a <label> for "Which facility?" dropdown option on patient bulk upload

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8572]
## Area or Component

`/app/upload-patients` aka "Add new patient" -> "Import from spreadsheet" option

## Quick Summary

The "Which facility?" dropdown option on the patient bulk upload page does not have a `<label>` associated with it instead it is marked up as a `<div>`.

### How to Replicate
1. Click "Patients"
2. Click "Add patients"
3. Click "Import from spreadsheet"
4. For the 2nd question, "Would you like to import these patients to one facility OR all facilities?", click "One facility."
5. The "Which facility?" text is not actually marked as a `<label>` for the dropdown below.

## Accessibility Violation

[label-not-present](https://docs.deque.com/issue-help/1.0.0/en/label-not-present)

## Action Requested

Remove the `<div>` that contains "Which facility?" and pass a `label` attribute for the dropdown and remove the `aria-label`.

## Screenshots (optional)

<img width="1336" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2tZQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--09840cb5d3ad25d9e250911318113acff92e7773/Screenshot%202025-03-03%20at%2016.22.46.png" alt="Screenshot 2025-03-03 at 16.22.46.png" />

## Source Code (optional)

https://github.com/CDCgov/prime-simplereport/blob/c862a94015ce46f67918b528ad3c89e7f7b4e75b/frontend/src/app/patients/UploadPatients.tsx#L429
